e2e_samples/parking_sensors/scripts/configure_databricks.sh

```diff
@@ -117,8 +117,23 @@ fi
 az keyvault secret set --vault-name "$KEYVAULT_NAME" --name "databricksClusterId" --value "$cluster_id" -o none
 
 
-log "Uploading libs TO dbfs..."
-databricks fs cp --recursive --overwrite "./databricks/libs/ddo_transform-localdev-py2.py3-none-any.whl" "dbfs:/ddo_transform-localdev-py2.py3-none-any.whl"
+# Find the generated .whl file dynamically
+WHL_FILE=$(find ./src/ddo_transform/dist/ -name "*.whl" | head -n 1)
+
+if [ -z "$WHL_FILE" ]; then
+    log "Error: No .whl file found in dist directory. Make sure the package is built." "error"
+    exit 1
+fi
+
+WHL_FILENAME=$(basename "$WHL_FILE")
+
+# Upload the dynamically found .whl file to Databricks
+log "Uploading $WHL_FILENAME to Databricks DBFS..."
+databricks fs cp --recursive --overwrite "$WHL_FILE" "dbfs:/$WHL_FILENAME"
```

Comment:
With moving to Databricks Unity Catalog, we are moving away from using DBFS. We need to update this with @ydaponte.